tests/test_commands.py

```diff
@@ -273,31 +298,52 @@ class MySpider(scrapy.Spider):
 
     def parse(self, response):
         return {'test': 'value'}
-""",
-                           args=('-s', 'DNSCACHE_ENABLED=False'))
-        print(log)
+"""
+        log = self.get_log(dnscache_spider, args=('-s', 'DNSCACHE_ENABLED=False'))
         self.assertNotIn("DNSLookupError", log)
         self.assertIn("INFO: Spider opened", log)
 
+        if sys.platform == 'win32':
+            log = self.get_log(dnscache_spider, name='myspider,pyw', args=('-s', 'DNSCACHE_ENABLED=False'))
+            self.assertNotIn("DNSLookupError", log)
+            self.assertIn("INFO: Spider opened", log)
+
     def test_runspider_log_short_names(self):
         log1 = self.get_log(self.debug_log_spider,
                             args=('-s', 'LOG_SHORT_NAMES=1'))
-        print(log1)
         self.assertIn("[myspider] DEBUG: It Works!", log1)
         self.assertIn("[scrapy]", log1)
         self.assertNotIn("[scrapy.core.engine]", log1)
 
         log2 = self.get_log(self.debug_log_spider,
                             args=('-s', 'LOG_SHORT_NAMES=0'))
-        print(log2)
```

Comment:
why we need these print statements?

Reply:
I don't think we need them, they can be useful when finding why a test fails but in that case you can add them locally

Reply:
Alright, I have already removed them. :)